pyvmc_csp.py

```diff
@@ -50,17 +50,31 @@ def get_csp_users_json(strCSPProdURL, orgID, session_token):
         print(json_response['error_message'])
 
 
-def get_csp_groups_json(strCSProdURL, org_id, session_token):
+def get_csp_groups_json(strCSProdURL, org_id, session_token,searchTerm):
+    """make the call to the API looking for groups that CONTAIN the search term - br"""
     myHeader = {'csp-auth-token': session_token}
-    myURL = f'{strCSProdURL}/csp/gateway/am/api/orgs/{org_id}/groups'
+
+    myURL = f'{strCSProdURL}/csp/gateway/am/api/orgs/{org_id}/groups-search?groupSearchTerm=' + searchTerm
```

Comment:
Don't need to change it now, but best practice is to use f strings for easier readability.  In a future push, can you include the variable `searchTerm` in with the rest of the f string?